mobile/src/components/course-module/components/task/components/task-notes/components/task-note-card/styles.ts

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from '~/common/enums/enums';
+
+const styles = StyleSheet.create({
+  container: {
+    marginBottom: 25,
+    backgroundColor: AppColor.BACKGROUND.GRAY_300,
+    padding: 20,
+    borderRadius: 10,
+  },
+  note: {
+    color: 'white',
+    fontSize: 16,
+  },
+  date: {
+    marginTop: 10,
+    color: AppColor.TEXT.GRAY_200,
+  },
+  textContainer: {
+    marginTop: 10,
+    justifyContent: 'space-between',
+  },
+  user: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  avatar: {
+    width: 30,
+    height: 30,
+    marginRight: 15,
+  },
+  name: {
+    color: '#C4C4C4',
```

Comment:
Don't we have this color?

Reply:
We don't) OR better set AppColor.TEXT.GRAY_100 ?
![image](https://user-images.githubusercontent.com/82529236/189044666-2fa635a1-5127-469a-b608-2a43ec0171e8.png)


![image](https://user-images.githubusercontent.com/82529236/189044096-bc27024d-afd4-41b9-ad8a-4ffceec97ee3.png)

Reply:
Yes, use color that we have